lib/core/src/animation/physics.rs

```diff
@@ -32,13 +32,7 @@ impl DragProperties {
 
 impl PhysicsForce for DragProperties {
     fn force(&self, kinematics:&KinematicsProperties) -> Vector3<f32> {
-        let velocity  = kinematics.velocity;
-        let speed     = velocity.norm();
-        if speed > 0.0 {
-            velocity.normalize() * speed * speed * -0.5 * self.coefficient
-        } else {
-            zero()
-        }
+        -kinematics.velocity * self.coefficient
```

Comment:
As we discussed via Discord, let's use here the correct equation (with `v*v`). In case of instability, I'd love to help. Of course, we can have the ability to choose different equasions, but we should test the physically correct one.